typhon/objects/networking/endpoints.py

```diff
@@ -321,3 +346,12 @@ def makeTCP4ServerEndpoint(port):
     """
 
     return TCP4ServerEndpoint(unwrapInt(port))
+
+
+@runnable(RUN_1)
+def makeTCP6ServerEndpoint(port):
```

Comment:
Has makeTCP6ServerEndpoint been added to the unsafe scope? Are there any tests of monte code using it?